lib-injection/sources/sitecustomize.py

```diff
@@ -333,21 +366,21 @@ def _inject():
                 # Also insert the bootstrap dir in the path of the current python process.
                 sys.path.insert(0, bootstrap_dir)
                 _log("successfully configured ddtrace package, python path is %r" % os.environ["PYTHONPATH"])
-                event = gen_telemetry_payload(
-                    [
-                        create_count_metric(
-                            "library_entrypoint.complete",
-                            [
-                                "injection_forced:" + str(runtime_incomp or integration_incomp).lower(),
-                            ],
-                        )
-                    ]
+                telemetry_data.append(
+                    create_count_metric(
+                        "library_entrypoint.complete",
+                        [
+                            f"injection_forced:{str(runtime_incomp or integration_incomp).lower()}",
+                        ],
+                    )
                 )
+                event = gen_telemetry_payload(telemetry_data)
                 send_telemetry(event)
             except Exception as e:
-                event = gen_telemetry_payload(
-                    [create_count_metric("library_entrypoint.error", ["error_type:" + type(e).__name__.lower()])]
+                telemetry_data.append(
+                    create_count_metric("library_entrypoint.error", [f"error_type:{type(e).__name__.lower()}"])
                 )
+                event = gen_telemetry_payload(telemetry_data)
```

Comment:
Probably just my lack of context/knowledge about telemetry, but doesn't this mean we keep adding more and more events to `telemetry_data` and then generate the telemetry payload with the expanding data? Should we be "resetting" the `telemetry_data` every time we send it?